package.json

```
@@ -8,12 +8,20 @@
   "license": "Apache",
   "scripts": {
     "api": "node src/api/index.js",
-    "seed": "node src/seedFake.js"
+    "seed": "node src/seedFake.js",
+    "scan:nft": "node src/scanner/scanNFTData.js",
+    "scan:old": "node src/scanner/nftOwnersAtBlock.js",
+    "seed:db": "node src/scanner/nftOwnersAtBlock.js && node src/scanner/scanNFTData.js"
```

nit: suggest some renamings
```suggestion
    "seed:fake": "node src/seedFake.js",
    "scan:syncState": "node src/scanner/<name>.js",
    "scan:start": "node src/scanner/scanNFTData.js",
```